packages/react-centra-checkout/src/index.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/export */
+
+export { default as ApiClient } from './ApiClient'
+export * from './ApiClient'
+
+export { default } from './Context'
```

Comment:
Wondering if it makes more sense or not to have all exports being non default exports here 🤔 @alexanderflink Thoughts?

Reply:
I think it's kinda nice to have default exports for each of them, don't we also have an eslint rule which requires a default export?

Reply:
Ahh, now I see what you mean. Totally makes sense. Will change that!